pkg/productinfo/alibaba/productinfo_alibaba.go

```diff
@@ -40,55 +68,14 @@ func NewAlibabaInfoer(regionId, accessKeyId, accessKeySecret string) (*AlibabaIn
 	}, nil
 }
 
-// Initialize downloads and parses the ECS price list on Alibaba Cloud
+// Initialize is not needed on Alibaba because price info is changing frequently
 func (e *AlibabaInfoer) Initialize() (map[string]map[string]productinfo.Price, error) {
```

Comment:
on demand prices are not changing frequently (they are not changing at all because of the static json we're using) so it doesn't make sense to parse it every ~4 minutes. That json parsing code should be here in the `initialize` method and the short lived info updater should update spot prices only.